book.rb

```diff
@@ -0,0 +1,9 @@
+class Book
+  attr_accessor :title, :author, :rentals
+
+  def initialize(title, author)
+    @title = title
+    @author = author
+    @rentals = []
+  end
+end
```

Comment:
- [ ] Please add a method for adding rentals here in the book class and the book itself should be sent as a parameter to create the new rental achieving the `has-many` association. 
[Hint] It will be something like that:
```ruby
def add_rental(person, date)
    Rental.new(date, self, person)
end
```